### ISSUE TITLE
Player Homes

### ISSUE THREAD

**Issue by ForgottenGlory** [ForgottenGlory/Living-Skyrim#143]
I've been asked occasionally about more player homes for LS. I don't see any harm in this, so I'll be investigating a few more tastefully chosen player homes for you to purchase and inhabit.

**Comment by doc8425** [comment]
Oldrim had a good one around Solitude, the Priory of the Cape, I believe was the name.

**Comment by ForgottenGlory** [comment]
I'm trying to stick to the same policy of no Oldrim mods if I can.

**Comment by doc8425** [comment]
Of course. Don't remember the name, but theres a little hideout between Riverwood and Whiterun. It's in a cave along the river bank. I'll get you the name when I get home.

**Comment by doc8425** [comment]
Rayek's End-SSE Expanded Edition

**Comment by Xillcy2** [comment]
I hope this is where I make my suggestion but here goes...

https://www.nexusmods.com/skyrimspecialedition/mods/5702

The patch for it here

https://www.nexusmods.com/skyrimspecialedition/mods/26912

Thank you 🙂
